modules/external-auth-disabling.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * authentication/external-auth.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="external-auth-disabling_{context}"]
+= Disabling direct authentication
+
+If necessary, you can disable direct authentication for your cluster and revert back to authenticating with the built-in OpenShift OAuth server.
+
+.Prerequisites
+
+* You have access to the `kubeconfig` file for the cluster.
+
+.Procedure
+
+. Log in to the cluster by using your `kubeconfig` file.
++
+You must use the `kubeconfig` file to configure the direct authentication. If you use a token, the token can become invalid during the process of the procedure.
+
+. Edit the authentication configuration by running the following command:
++
+[source,terminal]
+----
+$ oc edit authentication.config/cluster
+----
+
+. Update the authentication configuration by setting the `type` field to `IntegratedOAuth` and removing the `oidProviders` section:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Authentication
+metadata:
+# ...
```

Comment:
@liouk @xingxingxia I say to remove oidcProviders but don't show it in the example here. Do you think this is fine with it being gone, or do you think I should show it as commented out or anything?

Reply:
@bergerhoffer , to make it clear, it is better to make it shown for which you can replace oc edit with oc patch:
```
$ oc patch authentication.config/cluster --type=merge -p='
spec:
  type: ""                            <1>
  oidcProviders: null        <2>
'
```
Then say you want to say for <1> and <2>. For type: "", you can show "" (because the fresh env shows "") but say "" and "IntegratedOAuth" are same.

Reply:
Cool thanks that works. I don't mind using patch when they don't need to adjust any of the values before applying the command. Updating!